data/blog/opensats-receives-dollar250k-donation-from-tether.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: "OpenSats Receives $250,000 Donation From Tether"
+date: '2025-10-16'
+tags: ['OpenSats', 'funding', 'donations']
+authors: ['odell', 'dergigi']
+images: ['/static/images/blog/90-opensats-receives-donation-from-tether.jpg']
+draft: false
+summary: "Tether is supporting OpenSats with a $250,000 donation."
+---
+
+We are pleased to announce that [Tether](https://tether.io/), the largest
+company in the digital assets industry, has made a $250,000 donation to
+OpenSats. This contribution strengthens our ability to fund free and open-source
+projects that secure Bitcoin and advance the broader freedom tech ecosystem.
+
+As a 501(c)(3) public charity, OpenSats directs 100% of donations to our
+[General Fund](/funds/general) and [The Nostr Fund](/funds/nostr) to open-source
+projects and builders, while separate contributions to our [Operations
+Budget](/funds/ops) cover our administrative and operational expenses.
+
+Support from organizations like Tether helps us sustain our grant giving model
```

Comment:
```suggestion
Support from organizations like Tether helps us sustain our grant-giving model
```
---
From Grok:

when two or more words work together as an adjective before a noun, standard English grammar requires a hyphen to link them:

✓ a well-known author
✓ a decision-making process
✓ a grant-giving model
✓ a five-year plan
✓ a state-of-the-art facility